amberdata-gasprice/src/endpoint/gasprice.ts

```diff
@@ -0,0 +1,41 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+
+export const NAME = 'gasprice'
+
+const customError = (data: any) => {
+  return Object.keys(data.payload).length < 1
+}
+
+const customParams = {
+  speed: false,
+  endpoint: false,
```

Comment:
Kept this EA separate from `amberdata` until we have fully defined how we want to move forward with `endpoint` params. Currently conflicts with `amberdata`'s endpoint.